src/testData/testBeacons.ts

```diff
@@ -0,0 +1,56 @@
+import { BeaconStatuses, IBeacon } from "../entities/IBeacon";
+import { testUses } from "./testUses";
+import { testOwners } from "./testOwner";
+import { testEmergencyContacts } from "./testEmergencyContacts";
+import faker from "faker";
+
+export const testSingleBeacon: IBeacon = {
+  id: "f48e8212-2e10-4154-95c7-bdfd061bcfd2",
+  hexId: "1D0EA08C52FFBFF",
+  registeredDate: new Date("21 April 2021"),
+  status: BeaconStatuses.new,
+  manufacturer: "OceanSignal",
+  model: "EPIRB",
+  manufacturerSerialNumber: "123ABC",
+  chkCode: "456QWE",
+  batteryExpiryDate: new Date("1 April 2022"),
+  lastServicedDate: new Date("1 April 2019"),
+  uses: testUses,
+  owners: testOwners,
+  emergencyContacts: testEmergencyContacts,
+};
```

Comment:
Existing test data taken from `BeaconsGateway.testData.ts`.  As it's being used as test data for other classes/components, I've extracted it to its own `testData` file/folder structure.

Reply:
Same for `testEmergencyContacts` etc.